docs/core/spatial-rules.md

```diff
@@ -1,21 +0,0 @@
-# Spatial Rules
-
-Spatial rules allow you to create rules for certain areas. One example is that the default accessibility for 
-`highway=track` in Austria is `access=yes`, whereas the default accessibility is `access=destination` in Germany. Other 
-examples are different max speeds for different countries. More information on different road rules can be found in the 
-OSM wiki, different [accessibilities](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Access-Restrictions) and 
-different [speeds](https://wiki.openstreetmap.org/wiki/OSM_tags_for_routing/Maxspeed#Motorcar).
-
-## Enabling Rules
-
-If you have a working GraphHopper setup it is easy to enable Spatial Rules.
-We provide a set of approximate country borders, within the GraphHopper repository. If you need exact borders you can
-get the exact borders from [here](https://github.com/datasets/geo-countries). Go to your `config.yml` file and
```

Comment:
in `spatialrules/README.md` we describe how we got the borders from osm-boundaries.com. How is this different to the resource described here? And which one should we recommend? Should we still describe our borders as 'approximate' or are they more or less exact? @otbutz?

Reply:
> in `spatialrules/README.md` we describe how we got the borders from osm-boundaries.com. How is this different to the resource described here? And which one should we recommend?

I think i actually missed to update this file when i recreated our borders. The one referring to osm-boundaries is the correct one.

> Should we still describe our borders as 'approximate' or are they more or less exact? @otbutz?

They gained a lot more precision but should still be considered an approximation which should be sufficient for 99.9% of routing related decisions:

![grafik](https://user-images.githubusercontent.com/22315436/126970132-dd63bb03-53fb-4fbf-b38e-799e089b3902.png)

Reply:
Ok I think we can leave the documentation as it is then.